remo/voting/tasks.py

```diff
@@ -138,7 +141,7 @@ def create_rotm_poll():
                                     end__range=[start, end])
 
     if not now().date() > ROTM_NOMINATION_END_DATE or rotm_poll.exists():
```

Comment:
The code is not running because if the check here is true, then the 'enable_rotm_tasks' will be evaluated to false. The transaction will take place if the boolean flag is true or the switch is true, which is the behavior that we want.